android/java/org/chromium/chrome/browser/crypto_wallet/activities/BraveWalletActivity.java

```diff
@@ -169,32 +183,17 @@ protected void triggerLayoutInflation() {
         mCryptoWalletOnboardingViewPager.setOffscreenPageLimit(
                 mCryptoWalletOnboardingPagerAdapter.getCount() - 1);
 
-        ImageView onboardingBackButton = findViewById(R.id.onboarding_back_button);
-        onboardingBackButton.setVisibility(View.GONE);
-        onboardingBackButton.setOnClickListener(v -> {
-            if (mCryptoWalletOnboardingViewPager != null) {
-                mCryptoWalletOnboardingViewPager.setCurrentItem(
-                        mCryptoWalletOnboardingViewPager.getCurrentItem() - 1);
-            }
-        });
+        mOnboardingCloseButton = findViewById(R.id.onboarding_close_button);
+        mOnboardingCloseButton.setOnClickListener(v -> finish());
 
-        mCryptoWalletOnboardingViewPager.addOnPageChangeListener(
-                new ViewPager.OnPageChangeListener() {
-                    @Override
-                    public void onPageScrolled(
-                            int position, float positionOffset, int positionOffsetPixels) {}
-
-                    @Override
-                    public void onPageSelected(int position) {
-                        if (position == 0 || position == 2) {
-                            onboardingBackButton.setVisibility(View.GONE);
-                        } else {
-                            onboardingBackButton.setVisibility(View.VISIBLE);
-                        }
```

Comment:
This logic is not required anymore as now each fragment can be individually configured overriding the abstract methods `canBeClosed`, `canNavigateBack`.